components/ui/autocomplete/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <color name="colorAccent">#7f050082</color>
```

Comment:
Let's file a follow-up bug to get rid of this color definition here. It should be possible to either use the color from the accent app's theme (Let's investigate how appcompat does that) or make this configurable with an attribute in the layout XML where the component is used.

Reply:
OK, I've updated the commit to look up the color programmatically. So, this allows us to use whatever color the app defined as `colorAccent`.